### PR TITLE
Opiniated refactoring

### PR DESCRIPTION
**Pull request by christophgysin**

### CODE REVIEW

**Comment by christophgysin** [comment]
Feel free to ask about the reasons for any of the suggested changes, or discard this entirely :)

File: keybase_backup.py [hunk 1]
```diff
@@ -68,56 +57,51 @@ def download(message):
...
+    response = kb_call("read", options)
+
+    messages = response["messages"]
+    last = response["pagination"]["last"]
+    next_page = response["pagination"]["next"]
+
+    if last:
+        return messages
 
+    return messages + get_chat_history(conversation_id, next_page)
```
Comment (kisst) [inline keybase_backup.py:104]: E302: expected 2 blank lines, found 1

File: keybase_backup.py [hunk 2]
```diff
@@ -68,56 +57,51 @@ def download(message):
...
+            "pagination": {
+                "next": page,
+                "num": 1000,
+            }
+        }
+
+    response = kb_call("read", options)
+
+    messages = response["messages"]
+    last = response["pagination"]["last"]
```
Comment (kisst) [inline keybase_backup.py]: need to keep the `try`, as this key does not always exist, or just the `get_next` function

**Review by kisst — CHANGES_REQUESTED** [review]
All good, it make sense and the `kb_call` looks much better, but the functionality is currently broken due to the uncaught `KeyError`

File: keybase_backup.py (hunk 2 above)
Comment (christophgysin) [reply keybase_backup.py]: My keybase chat history is not big enough to trigger pagination it seems :)

No need to throw/catch a KeyError though, we can either check for the presence of the key, or `.get()` it with a default value as fallback.

**Review by kisst — APPROVED** [review]
That works too, I knew there must be a better way...

**Pull request merged**